packages/components/.storybook/decorators/LayoutDecorator.tsx

```diff
@@ -0,0 +1,34 @@
+import { DecoratorFunction } from '@storybook/addons';
```

Comment:
Also add `@storybook/addons` to `devDependencies`

Reply:
Was already installed I think in another package

Reply:
Since it's used in this package, we should add it to the `devDependencies` from this package too to be correct.

Reply:
Added